dojo/db_migrations/0123_sonarqube_api_type_length_change.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.17 on 2021-03-12 14:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0122_cobaltio_product'),
```

Comment:
```suggestion
        ('dojo', '0123_scan_type'),
```

There has been another migration merged in dev

Reply:
@jimtsikos Will merge it directly when this is resolved 😃

Reply:
@StefanFl Changes applied.